src/spikeinterface/core/tests/test_node_pipeline.py

```diff
@@ -0,0 +1,186 @@
+import pytest
+import numpy as np
+from pathlib import Path
+import shutil
+
+import scipy.signal
```

Comment:
@samuelgarcia you need to add `scipy` in `test_core` extras!

Reply:
I removed scipy instead